src/app/home/layouts/budgets-form/budgets-form.component.ts

```diff
@@ -7,9 +8,14 @@ import { Component, OnInit } from '@angular/core';
 })
 export class BudgetsFormComponent implements OnInit {
 
-  constructor() { }
+  constructor(private initialSettingsService: InitialSettingsService) {
+  }
 
-  ngOnInit(): void {
+  ngOnInit() {
+    this.sendCurrentStepInfo();
   }
 
+  sendCurrentStepInfo() {
+    this.initialSettingsService.sendCurrentStepInfo('step3');
```

Comment:
Is there a need for keeping the step number as a string? I think simple number would be just enough for that.